src/app/users/brokering/broker-form.component.html

```diff
@@ -0,0 +1,24 @@
+<page-layout [ready]="data$ | async">
+  <ng-container *ngIf="data">
+    <page-content [heading]="i18n.brokers.title.new"
+      [mobileHeading]="i18n.brokers.mobileTitle.new">
+      <form [formGroup]="form">
+        <user-field [filters]="{
```

Comment:
A cosmetic suggestion is to change the full filters to a property, so the html is a bit cleaner.